sdl2-vga-terminal/test/valgrind/helgrind.cpp

```diff
@@ -0,0 +1,134 @@
+#include <iostream>
+#include <VgaTerminal.hpp>
+#include <cstdlib>
+
+using namespace std;
+
+int main(int argc, char* args[])
+{
+	if (SDL_Init(SDL_INIT_VIDEO | SDL_INIT_TIMER | SDL_INIT_EVENTS) != 0) {
+		SDL_LogError(SDL_LOG_CATEGORY_SYSTEM, "Unable to initialize SDL: %s", SDL_GetError());
+		return 1;
+	}
+	// --- end SDL2 system init and info
+
+	VgaTerminal term1 = VgaTerminal("VgaTerminal1", 0, 0, 640, 400, 0, -1, 0);
+	VgaTerminal term2 = VgaTerminal("VgaTerminal2", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, 640, 400, 0, -1, 0);
+
+	
+	term1.cursor_time = 100;
+	term2.cursor_time = 100;
+
+	for (int i = 0; i < 256; i++) {
+		term1.write(static_cast<uint8_t>(i), i, 255 - i);
+		term2.write(static_cast<uint8_t>(i), i, 255 - i);
+	}
+
+	term1.render();
+	term1.writeXY(10, 15, "���������ͻ", 14, 2);
+	term1.writeXY(10, 16, "�         �", 14, 2);
+	term1.writeXY(10, 17, "���������ͼ", 14, 2);
+	term1.gotoXY(12, 16); term1.write(3, 4, 15);
+	term1.gotoXY(14, 16); term1.write(4, 15, 4);
+	term1.gotoXY(16, 16); term1.write(5, 0, 15);
+	term1.gotoXY(18, 16); term1.write(6, 15, 0);
+	term1.render();
+	term1.gotoXY(40, 24);
+	
+	term2.writeXY(40, 10, "Terminal II", 7, 0);
+
+	
+	term1.render();
+	term2.render();
+	
+	SDL_Event e;
+	for (int i = 0; i < 10;) {
+		SDL_WaitEvent(&e);
+		if (e.window.windowID == term1.getWindowId()) {
+			term1.render();
+			i++;
+		}
+		else if (e.window.windowID == term2.getWindowId()) {
+			term2.render();
+			i++;
+		}
+		else {
+			// ignoring
+		}
+	}
+	
+	srand(time(0));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [time is Y2038-unsafe](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5427759)